packages/devtools_app/lib/src/debugger/html_debugger_screen.dart

```diff
@@ -414,6 +414,8 @@ class HtmlDebuggerScreen extends HtmlScreen {
 
   @override
   void onContentAttached() {
+    // Enable this page to display in a shadow DOM from Flutter.
+    assert(overrideDocumentRoot.isConnected);
```

Comment:
have you tested to make sure the dart:html version of the app still works with this change?

Reply:
The integration tests have :smile: 

https://travis-ci.org/flutter/devtools/jobs/602574188#L714